tests/__init__.py

```diff
@@ -57,11 +59,15 @@ def test_works_as_expected(self):
                     tag_elements = tag_elements[1:]
 
                 headers = {
-                    "kind": tag_elements[0]
+                    "kind": tag_elements[0],
+                    'stage': "recognized",
                 }
                 if len(tag_elements) > 1:
                     if tag_elements[0] == "archive":
                         headers["extension"] = tag_elements[1]
+                    elif (tag_elements[0] == "unknown"):
```

Comment:
In the meantime I found these unnecessary braces in test:
```suggestion
                    elif tag_elements[0] == "unknown":
```